### ISSUE TITLE
Create better examples

### ISSUE THREAD

**Issue by jintonic** [jintonic/shine#13]
replace the original three.js examples with better once from <https://github.com/OpenGATE/GateContrib>

**Comment by jintonic** [comment]
@bestdeveloper23, please work on this first

**Comment by bestdeveloper23** [comment]
I can find good geometry about human brain, skull, etc..
So I should add these examples to left panel?

**Comment by jintonic** [comment]
No. There is a menu entry called "Examples". Remove the existing one there, and add new ones in the same menu.

**Comment by bestdeveloper23** [comment]
I see.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/lgWq9SrU9nUECfPgatrYbY2WjOooWA
https://monosnap.com/file/eybgGZ2baI65PonUSRJdBUONRC2zcp

I found human brain and skull.

**Comment by jintonic** [comment]
Looks very nice, please add them in

**Comment by bestdeveloper23** [comment]
Yes.

**Comment by bestdeveloper23** [comment]
I added 4 examples.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/SoRYk0ygXbe8Ut1CmJnpy7PdkiZW9D
It works well on my localhost.

**Comment by jintonic** [comment]
Please remove the original examples. How many hours did you work on this?

**Comment by bestdeveloper23** [comment]
It took 4 hours, but now a 404 error occurs on the live server.
So I'm working on solving that.

**Comment by bestdeveloper23** [comment]
Yes.
It was easy.
But I'm feeling stressed now.

**Comment by jintonic** [comment]
All right, I trust you. So you have worked for 8 hours now (4 h for GDML and 4 h for examples). The live server works for me. 
- [ ] But the skull and brain point to the same object. 
- [ ] A refresh function is missing as well. I need to click the canvas to see the placed example.
- [ ] When I select monkey and kidney, the objects are not added to the "scene" list in the right panel.

**Comment by bestdeveloper23** [comment]
I fixed all.
What's the next?

**Comment by jintonic** [comment]
Thank you so much! Please work on "export to text geometry"
